### ISSUE TITLE
Formatter deleting CQL with inline comment

### ISSUE THREAD

**Issue by jackcmeyer** [cqframework/clinical_quality_language#215]
The following CQL has parts of it being deleted when going through the formatter: 

```
library Test version '0.0.000'

using QDM version '5.3'

valueset "Dapsone and pyrimethamine": 'urn:oid:2.16.840.1.113883.3.464.1003.196.12.1202' version 'urn:hl7:version:eCQM%20Update%202017-05-05'
valueset "Leucovorin": 'urn:oid:2.16.840.1.113883.3.464.1003.196.12.1205' version 'urn:hl7:version:eCQM%20Update%202017-05-05'

parameter "Measurement Period" Interval<DateTime>

context Patient

define "test":
	["Medication, Order": "Dapsone and pyrimethamine"] MO
	// some comment
		  let var: 2
```

Results in: 

```
library Test version '0.0.000'

using QDM version '5.3'

valueset "Dapsone and pyrimethamine": 'urn:oid:2.16.840.1.113883.3.464.1003.196.12.1202' version 'urn:hl7:version:eCQM%20Update%202017-05-05'
valueset "Leucovorin": 'urn:oid:2.16.840.1.113883.3.464.1003.196.12.1205' version 'urn:hl7:version:eCQM%20Update%202017-05-05'

parameter "Measurement Period" Interval<DateTime>

context Patient

define "test":
	["Medication, Order": "Dapsone and pyrimethamine"] MO
	// some comment
		   let var:
```

(note the missing 2 after var: )

**Comment by brynrhodes** [comment]
Hi @jackcmeyer, was this issue found with the latest code from master, or with the latest published translator package?

**Comment by jackcmeyer** [comment]
Hi @brynrhodes, this is with the most recent master branch code.

**Comment by brynrhodes** [comment]
Addressed in v1.2.12-SNAPSHOT.
